test/integration/targets/collections/import_collection_pb.yml

```diff
@@ -0,0 +1,2 @@
+- import_playbook: testns.testcoll.default_collection_playbook.yml
+- import_playbook: testns.testcoll.default_collection_playbook
```

Comment:
I'd argue that we should only support just this case :point_up_2:

Reply:
easy enough to revert but i expect there to be user pushback, since there already was from the small sample that tested this PR initially.